connector/doc/guides/jobrunner.rst

```diff
@@ -19,13 +19,20 @@ The ``ODOO_CONNECTOR_CHANNELS`` environment variable must be
 set before starting Odoo in order to enable the job runner
 and configure the capacity of the channels.
 
+Alternatively, set the channel configuration in the Odoo configuration file:
+
+.. code-block:: cfg
+
+    [options-connector]
+    channels = root:4
```

Comment:
Hi @pedrobaeza,

To make your proposed syntax work I had to add whitespace handling to the channel configuration parsing. Should be backward compatible.

So yes, your proposed syntax should work now, and it even allows for trailing spaces and commented lines (which are discarded by the config parser and manifest as empty entries).